mdbenchmark/submit.py

```diff
@@ -75,39 +85,51 @@ def submit(directory, force_restart):
-
-    if not bundles_not_yet_started and not force_restart:
-        console.error(
-            "All generated benchmarks were already started once. "
-            "You can force a restart with {}.",
-            "--force",
+    df = MakeDataFrameFromBundle(bundle)
+    # Reformat NaN values nicely into question marks.
+    df_to_print = df.replace(np.nan, "?")
+    df_to_print = df.drop(columns=['ns/day','ncores'])
+    with pd.option_context("display.max_rows", None):
+        print(df_to_print)
+
+    # here I add the user promt to confirm the submission of the simulations
+    if yes or click.confirm('The above simulations will be submitted. Continue?'):
+        grouped_bundles = bundle.categories.groupby("started")
+        try:
+            bundles_not_yet_started = grouped_bundles[False]
+        except KeyError:
+            bundles_not_yet_started = None
+
+        if not bundles_not_yet_started and not force_restart:
+            console.error(
+                "All generated benchmarks were already started once. "
+                "You can force a restart with {}.",
+                "--force",
+            )
+
+        # Start all benchmark simulations if a restart was requested. Otherwise
+        # only start the ones that were not run yet.
+        bundles_to_start = bundle
+        if not force_restart:
+            bundles_to_start = bundles_not_yet_started
+
+        batch_cmd = get_batch_command()
+
+        console.info("Submitting a total of {} benchmarks.", len(bundles_to_start))
+
+        for sim in bundles_to_start:
+            # Remove files generated by previous mdbenchmark run
+            if force_restart:
+                engine = detect_md_engine(sim["module"])
+                cleanup_before_restart(engine=engine, sim=sim)
+
+            sim.categories["started"] = True
+            os.chdir(sim.abspath)
+            subprocess.call([batch_cmd, "bench.job"])
+
+        console.info(
+            "Submitted all benchmarks. Run {} once they are finished to get the results.",
+            "mdbenchmark analyze",
         )
-
-    # Start all benchmark simulations if a restart was requested. Otherwise
-    # only start the ones that were not run yet.
-    bundles_to_start = bundle
-    if not force_restart:
-        bundles_to_start = bundles_not_yet_started
-
-    batch_cmd = get_batch_command()
-    console.info("Submitting a total of {} benchmarks.", len(bundles_to_start))
-
-    for sim in bundles_to_start:
-        # Remove files generated by previous mdbenchmark run
-        if force_restart:
-            engine = detect_md_engine(sim["module"])
-            cleanup_before_restart(engine=engine, sim=sim)
-
-        sim.categories["started"] = True
-        os.chdir(sim.abspath)
-        subprocess.call([batch_cmd, "bench.job"])
-
-    console.info(
-        "Submitted all benchmarks. Run {} once they are finished to get the results.",
-        "mdbenchmark analyze",
-    )
+    else:
```

Comment:
This and the following line can be removed, when the if-clause above is inverted (see previous comment).